src/app/real-estate-management/real-estate-management.component.html

```diff
@@ -1,22 +1,23 @@
 <div id="container" class="d-flex flex-column gap-3 justify-content-center align-items-center">
   <h3 id="page-title">Real Estate Management</h3>
   <section class="w-75">
-    <button class="btn btn-primary w-100" id="button-add-real-estate" type="button"
-      data-target="#exampleModal" (click)="openModal()"><i class="bi bi-plus"></i> Add Real Estate</button>
+    <button class="btn btn-primary w-100" id="button-add-real-estate" type="button" (click)="openModal()">
+      <i class="bi bi-plus"></i> Add Real Estate
+    </button>
     <div class="accordion accordion-flush" id="accordionFlushExample">
-      <div class="accordion-item" *ngFor="let realEstateHeader of realEstatesHeaderList$ | async">
+      <div class="accordion-item" *ngFor="let realEstateHeader of realEstatesHeaderList">
         <h2 class="accordion-header">
-          <button class="accordion-button collapsed" type="button" (click)="getRealEstateBody(realEstateHeader.id)" data-bs-toggle="collapse" [attr.data-bs-target]="'#flush-collapseThree' + realEstateHeader.id" aria-expanded="false" aria-controls="flush-collapseThree">
-            {{realEstateHeader.typologyId}} - {{realEstateHeader.cityId}} I {{realEstateHeader.title}}
+          <button class="accordion-button collapsed" type="button" (click)="getRealEstateBody(realEstateHeader.id)" data-bs-toggle="collapse" [attr.data-bs-target]="'#flush-collapse' + realEstateHeader.id" aria-expanded="false">
```

Comment:
If we have ngIf do we need this: data-bs-toggle="collapse"

Reply:
> If we have ngIf do we need this: data-bs-toggle="collapse"
Doesn't work without data-bs-toggle="collapse"